README.md

```diff
@@ -1,10 +1,41 @@
-canillita
-=========
+#canillita
 
-Simple Paperboy-themed PubSub
+Simple Paperboy-themed PubSub on top of REST+SSE.
 
 You can find a blog post about this app somewhere in [Inaka's Blog](http://inaka.net/blog) :)
 
 ## :warning: DISCLAIMER :warning:
-
 This is canillita **v2**, if you're looking for **v1** switch to branch [v1](https://github.com/inaka/canillita/tree/v1) or tag [1.0](https://github.com/inaka/canillita/tree/1.0) :)
+
+### Introduction
+**Canillita** is a RESTful API that allow us to manage `news` by `newspaper` and every time a `news item` is created it gets notified to the listeners.
```

Comment:
is a RESTful API -> provides a RESTful API
is created it ... listeners -> is published, the listeners get a notification with it.